smarts/env/run_env.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import argparse
+import gym
+
+from smarts.core.utils.episodes import episodes
+
+logging.basicConfig(level=logging.ERROR)
+
+
+def main(scenarios, headless, num_episodes, max_episode_steps=None):
+    env = gym.make(
+        "smarts.env:hiway-v0",
+        scenarios=scenarios,
+        agent_specs={},
+        headless=headless,
+        sumo_headless=True,
+    )
+
+    if max_episode_steps is None:
+        max_episode_steps = 1000
```

Comment:
I think this might need some configuration (although ideally I think this max step information should also be inherent in the scenario.)